runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcIndicatorsConfig.java

```diff
@@ -107,8 +109,22 @@ default Color getHighlightColor()
 		return Color.CYAN;
 	}
 
+	@Range(
+		max = MAX_STROKE_SIZE
+	)
 	@ConfigItem(
 		position = 5,
+		keyName = "strokeSize",
+		name = "Outline Width",
+		description = "Width of the NPC highlight"
+	)
+	default int getStrokeSize()
```

Comment:
This can follow the same example as d4078a95553bee262f0d5ca032cea49dfca5ca3e and be a `double` config.